src/pybind/mgr/requirements-required.txt

```diff
@@ -14,3 +14,4 @@ requests-mock
 scipy
 werkzeug
 natsort
+bcrypt
```

Comment:
to avoid any unintentional breaking, isn't it good to pin them with a version.

Reply:
> to avoid any unintentional breaking, isn't it good to pin them with a version.

@nizamial09 I added the dependency here because I copied the one from dashboard. Unfortunately the original one wasn't pinned to a specific version. Which version should I fix here?

Reply:
since the latest version hasn't failed yet, we can take the latest itself I suppose. latest version looks like [4.0.1](https://pypi.org/project/bcrypt/#history), or we can leave it as it is too. I'll leave that up to you.

Reply:
@nizamial09 well, considering this special case (a security library) I think it's better to go with the latest so we don't miss any critical patches if any.